tool/flutter_ci_script_master.sh

```diff
@@ -21,7 +21,8 @@ declare -ar PROJECT_NAMES=(
     "desktop_photo_search/material"
     "experimental/context_menus"
     "experimental/federated_plugin/federated_plugin"
-    "experimental/web_dashboard"
+    # TODO(DomesticMouse): Error: Method not found: 'FallThroughError'.
```

Comment:
It sounds like one of the switch/case statements falling through, but this error has been [deprecated since 2.0](https://api.flutter.dev/flutter/dart-core/FallThroughError-class.html) so I'm not sure what's going on here.

Reply:
I'll worry about it a bit more when it makes it to beta